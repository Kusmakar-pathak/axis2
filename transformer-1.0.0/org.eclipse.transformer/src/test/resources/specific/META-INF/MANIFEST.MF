Manifest-Version: 1.0
Bundle-Name: Specific
Bundle-SymbolicName: specific
DynamicImport-Package: javax.servlet;version="4.0.0",javax.servlet.ann
 otation;version="4.0.0",javax.servlet.http;version="4.0.0",javax.anno
 tation.security;version="4.0.0"
Export-Package: javax.servlet;version="4.0.0",javax.servlet.annotation
 ;version="4.0.0",javax.servlet.http;version="4.0.0",javax.annotation.
 security;version="4.0.0"
Import-Package: javax.servlet;version="4.0.0",javax.servlet.annotation
 ;version="4.0.0",javax.servlet.http;version="4.0.0",javax.annotation.
 security;version="4.0.0"

