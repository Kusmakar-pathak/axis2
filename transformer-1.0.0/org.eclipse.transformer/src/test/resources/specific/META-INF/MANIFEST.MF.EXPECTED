Manifest-Version: 1.0
Bundle-Name: Specific Jakarta
Bundle-SymbolicName: specific.jakarta
DynamicImport-Package: jakarta.servlet;version="[2.6,6.0)",jakarta.se
 rvlet.annotation;version="[2.6,6.0)",jakarta.servlet.http;version="4
 .0.0",jakarta.annotation.security;version="4.0.0"
Export-Package: jakarta.servlet;version="6.0",jakarta.servlet.annotat
 ion;version="6.0",jakarta.servlet.http;version="6.0",jakarta.annotat
 ion.security;version="4.0.0"
Import-Package: jakarta.servlet;version="5.0",jakarta.servlet.annotat
 ion;version="[2.6,6.0)",jakarta.servlet.http;version="6.0",jakarta.a
 nnotation.security;version="4.0.0"

